contracts/MemeBase.sol

```diff
@@ -150,7 +150,7 @@ contract MemeBase is MemeFactory {
     /// @param olasAmount OLAS amount.
     /// @param tokenGasLimit Token gas limit for bridging OLAS to L1.
     /// @return msg.value leftovers if partially utilized by the bridge.
-    function _bridgeAndBurn(uint256 olasAmount, uint256 tokenGasLimit, bytes memory) internal override returns (uint256) {
```

Comment:
This are required essential changes that should have been implemented originally.

Reply:
Making functions virtual allows to inherit further and further. This is a minimal change so no re-deployment for the Base one is needed.